lib/tabular_form_builder.rb

```diff
@@ -70,24 +54,107 @@ def #{selector}(field, options = {})
   end
 
   def select(field, choices, options = {}, html_options = {})
-    label_for_field(field, options) + super
+    html_options[:class] = Array(html_options[:class]) + %w(form--select)
+
+    label_for_field(field, options) + container_wrap_field(super, 'select', options)
   end
 
   def collection_select(field, collection, value_method, text_method, options = {}, html_options = {})
-    label_for_field(field, options) + super
+    html_options[:class] = Array(html_options[:class]) + %w(form--select)
+
+    label_for_field(field, options) + container_wrap_field(super, 'select', options)
+  end
+
+  def collection_check_box(field,
+                           value,
+                           checked,
+                           text = field.to_s + "_#{value}",
+                           options = {})
+
+    label_options = options.reverse_merge(label: text)
+
+    label_for = "#{field}_#{value}".to_sym
+    label = label_for_field(label_for, label_options)
+
+    input_options = options.merge(multiple: true, no_label: true, checked: checked)
+    input = check_box(field, input_options, value, '')
+
+    label + container_wrap_field(input, 'check-box', options)
```

Comment:
@ulferts unfortunately, it looks like checkboxes are being wrapped in a container twice here

![seeded_project_-_settings_-_openproject](https://cloud.githubusercontent.com/assets/755/6107432/26c04c98-b06b-11e4-9621-f8d8d0557fa9.png)